groovy/inductor_dashboard.groovy

```diff
@@ -0,0 +1,376 @@
+            '''
+        } 
+    }
+    stage('login container & prepare scripts & run') {
+        echo 'login container & prepare scripts & run......'
+        if ("${isOP}" == "true") {
+            sh '''
+            #!/usr/bin/env bash
+            tag=${image_tag}
+            docker run -tid --name op_pt_inductor --privileged --env https_proxy=${https_proxy} --env http_proxy=${http_proxy} --net host  --shm-size 1G -v ${WORKSPACE}/opbench_log:/workspace/pytorch/dynamo_opbench ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag}
+            docker cp scripts/microbench/microbench_parser.py op_pt_inductor:/workspace/pytorch
+            docker cp scripts/microbench/microbench.sh op_pt_inductor:/workspace/pytorch
+            docker exec -i op_pt_inductor bash -c "bash microbench.sh dynamo_opbench ${op_suite} ${op_repeats} ${DT};cp microbench_parser.py dynamo_opbench;cd dynamo_opbench;pip install openpyxl;python microbench_parser.py -o ${_VERSION} -l ${BUILD_URL} -n ${_NODE};rm microbench_parser.py"
+            '''
+        }else {
+            sh '''
+            #!/usr/bin/env bash
+            tag=${image_tag}
+            docker run -tid --name pt_inductor --privileged --env https_proxy=${https_proxy} --env http_proxy=${http_proxy} --net host  --shm-size 1G -v ${WORKSPACE}/inductor_log:/workspace/pytorch/inductor_log ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag}
+            docker cp scripts/modelbench/inductor_test.sh pt_inductor:/workspace/pytorch         
+            docker cp scripts/modelbench/log_parser.py pt_inductor:/workspace/pytorch           
+            docker exec -i pt_inductor bash -c "bash inductor_test.sh ${THREAD} ${CHANNELS} ${DT} ${SHAPE} inductor_log"
+            '''
+        }
+    }
+    stage('archiveArtifacts') {
+        if ("${isOP}" == "true"){
+            archiveArtifacts artifacts: "**/opbench_log/**", fingerprint: true
+        }else{
+            archiveArtifacts artifacts: "**/inductor_log/**", fingerprint: true
+        }
+    }
+    stage("Sent Email"){
+        if ("${debug}" == "true"){
+            maillist="yudong.si@intel.com"
+        }else{
+            maillist="Chuanqi.Wang@intel.com;guobing.chen@intel.com;beilei.zheng@intel.com;xiaobing.zhang@intel.com;xuan.liao@intel.com;Chunyuan.Wu@intel.com;Haozhe.Zhu@intel.com;weiwen.xia@intel.com;jiong.gong@intel.com;eikan.wang@intel.com;fan.zhao@intel.com;shufan.wu@intel.com;weizhuo.zhang@intel.com;yudong.si@intel.com;diwei.sun@intel.com"
+        }
+        if (fileExists("${WORKSPACE}/opbench_log/op-microbench-${_VERSION}.xlsx") == true){

```

Comment:
Rethink about the email report, can dynamic generate report content according to specific test items, for example, if we specific the `OPBench` & `LLMBench` at the same time, the report should include those 2 part test results.

Reply:
It should be possible. I will try it, thanks